addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.scss

```diff
@@ -78,17 +73,17 @@ $o-mail-DiscussSidebarChannel-borderOpacity: .05;
     left: 3px;
 }
 
-.o-mail-DiscussSidebarChannel-subChannel.o-nonCompact {
+.o-mail-DiscussSidebarChannel-subChannel {
     margin-left: map-get($spacers, 4) + map-get($spacers, 2);
 }
 
 .o-mail-DiscussSidebar-unreadIndicator {
     font-size: 0.35rem;
-    left: 0px;
+    left: 1px;
+}
 
-    .o-mail-DiscussSidebarSubchannel &:not(.o-compact) {
-        left: map-get($spacers, 2) + map-get($spacers, 1) / 2;
-    }
+.o-mail-DiscussSidebarSubchannel .o-mail-DiscussSidebar-unreadIndicator {
+    left: map-get($spacers, 2);
```

Comment:
Same remark: value is `left: map-get($spacers, 2) + map-get($spacers, 1) / 2;`, should keep it that way.